classes/module/Module.php

```diff
@@ -28,8 +28,9 @@
 use PrestaShop\PrestaShop\Adapter\Module\ModuleDataProvider;
 use PrestaShop\PrestaShop\Core\Module\WidgetInterface;
 use PrestaShop\PrestaShop\Adapter\ServiceLocator;
+use PrestaShop\PrestaShop\Core\Module\ModuleInterface;
 
-abstract class ModuleCore
+abstract class ModuleCore implements ModuleInterface
```

Comment:
Why do we add this interface? If the legacy class is supposed to disappear, I do not understand this change.